markdown-mode.el

```diff
@@ -278,6 +278,16 @@ This is the default search behavior of Ikiwiki."
 (make-obsolete-variable 'markdown-wiki-link-search-subdirectories 'markdown-wiki-link-search-type "2.5")
 (make-obsolete-variable 'markdown-wiki-link-search-parent-directories 'markdown-wiki-link-search-type "2.5")
 
+(make-obsolete 'markdown-match-wiki-link nil "2.3")
```

Comment:
The 3rd argument of the `make-obsolete` means that **when** this function was made obsolete. So, it should be next release version(`2.8`), not past released versions